com.meetfranz.Franz.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "com.meetfranz.Franz",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "franz",
+    "separate-locales": false,
+    "rename-desktop-file": "franz.desktop",
+    "rename-icon": "franz",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "franz-desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x franz_*.deb",
+                "rm -f franz_*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "chmod -R a-s,go+rX,go-w /app/Franz",
+                "sed -i 's|Exec=.*|Exec=franz %U|' /app/share/applications/franz.desktop",
```

Comment:
You can use `desktop-file-utils` here

Reply:
ah. this is carried over from `org.signal.Signal`

Reply:
example?

Reply:
https://github.com/flathub/com.github.z.Cumulonimbus/blob/785266a8ec0c2ebb73eea05571518238e193eb0c/com.github.z.Cumulonimbus.json#L32